### ISSUE TITLE
JAWS and Application Role

### ISSUE THREAD

**Issue by terracoda** [phetsims/a11y-research#59]
From JAWS documentation:
[http://www.freedomscientific.com/Support/TechnicalSupport/Bulletin/1404](http://www.freedomscientific.com/Support/TechnicalSupport/Bulletin/1404)

> Application role within a Web page:
> 
> When JAWS encounters an aria application embedded within a Web page (i.e., the role of "application" is set on a descendent of the body element), JAWS slightly changes the behavior of Auto Forms Mode. These changes only take effect for those form controls that are descendents of the container element with the role of "application".
> 
> Note: JAWS actually only changes the logic which would normally cause it to exit Forms Mode. The process of entering Forms Mode remains the same.
> 
>    Once Forms Mode is on, **JAWS will pass the ARROW keys through to the application (my emphasis)**, and will not leave Forms Mode automatically based on arrow movement. This means that navigation of the application area must be done using the TAB key, and ARROW keys always manipulate the focused control.

@jessegreenberg, my reading of this JAWS documentation is that the Arrow keys should work (i.e., should be passed through to the application, i.e., should be passed through to move something like a balloon) when the Application role is used. 

Is that your reading of this documentation?

**Comment by jessegreenberg** [comment]
@terracoda totally agree.  The problem is that JAWS mangles the arrow key events that are passed to the application, making it impossible to detect when a key is pressed down and when it is released. That is why we submitted a bug report in #9.

**Comment by terracoda** [comment]
OK, got it! Thanks for clarification. Closing this issue.
